src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -58,15 +58,15 @@
 
 "ExposureDetection_Low" = "Low Risk";
 
-"ExposureDetection_High" = "Higher Risk";
+"ExposureDetection_High" = "Increased Risk";
 
 "ExposureDetection_NumberOfDaysStored" = "%d of 14 days active";
 
 "ExposureDetection_Refreshed" = "Updated: %@";
 
 "ExposureDetection_Refreshed_Never" = "Unknown";
 
-"ExposureDetection_RefreshingIn" = "Update in %02d:%02d\U00A0minutes";
+"ExposureDetection_RefreshingIn" = "Update in %02d:\U00A0minutes";
```

Comment:
Can someone explain that change?

even assuming you want to remove displaying unnecessary information: why remove the `%02d` but then keeping the `:` ?

Note that this change is also in the other translations of this PR.

For german see:

https://github.com/corona-warn-app/cwa-app-ios/blob/develop/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L69

https://github.com/corona-warn-app/cwa-app-ios/blob/95e7ea31ba48b92db3f72d55eefbccbb3a5bb98e/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L69

Reply:
good catch...